internal/authorizedapp/database/authorized_app_test.go

```diff
@@ -134,29 +135,31 @@ func TestGetAuthorizedApp(t *testing.T) {
 		{
 			name: "bare",
 			sql: `
-				INSERT INTO AuthorizedApp (app_package_name, platform, allowed_regions)
-				VALUES ($1, $2, $3)
+				INSERT INTO AuthorizedApp (app_package_name, platform, allowed_regions, health_authority_ids)
+				VALUES ($1, $2, $3, $4)
 			`,
-			args: []interface{}{"myapp", "android", []string{"US"}},
+			args: []interface{}{"myapp", "android", []string{"US"}, []int64{1}},
 			exp: &model.AuthorizedApp{
 				AppPackageName:           "myapp",
 				Platform:                 "android",
 				AllowedRegions:           map[string]struct{}{"US": {}},
+				AllowedHealthAuthorities: map[int64]struct{}{1: {}},
 				SafetyNetBasicIntegrity:  true,
 				SafetyNetCTSProfileMatch: true,
 			},
 		},
 		{
 			name: "all_regions",
 			sql: `
-				INSERT INTO AuthorizedApp (app_package_name, platform, allowed_regions)
-				VALUES ($1, $2, $3)
+				INSERT INTO AuthorizedApp (app_package_name, platform, allowed_regions, health_authority_ids)
```

Comment:
This mismatch between the table column and struct field is a bit weird.

Reply:
aligned the names... by changing both.